### DISCUSSION TITLE
`nargo fmt --check`

### DISCUSSION THREAD

**Discussion by benesjan** [noir-lang/noir#3519] [Ideas]
### Problem

We would like to integrate `nargo fmt`  in CI in aztec-packages but to do that we would need to run `nargo fmt` in "check-only mode" (not modifying the files and just throwing instead).**

### Happy Case

`nargo fmt --check` exits with exit code 1 when formatting issues are spotted.

### Would you like to submit a PR for this Issue?

No

**Comment by Savio-Sou** [comment]
Converted into https://github.com/noir-lang/noir/issues/3520.
